pkg/controller/kogitodataindex/resource/create_resources.go

```diff
@@ -84,7 +88,8 @@ func CreateOrFetchResources(instance *v1alpha1.KogitoDataIndex, context resource
 		build(createProtoBufConfigMap).
 		build(createStatefulSet).
 		build(createService).
-		buildOnOpenshift(createRoute)
+		buildOnOpenshift(createRoute).
```

Comment:
What is this '.' EOL ?

Reply:
`buildOnOpenshift(createRoute)` returns `*kogitoDataIndexResourcesFactory`. We want to do `*kogitoDataIndexResourcesFactory.build(createKafkaTopic)` after `buildOnOpenshift(createRoute)`.  Here, `.` is used to [chain the calls](https://en.wikipedia.org/wiki/Method_chaining).